.travis.yml

```diff
@@ -6,9 +6,26 @@ python:
   - "3.6"
   - "pypy3.5"
 # command to install dependencies
-install: "pip install -r requirements.txt"
+install:
```

Comment:
Thanks for putting this together @c-goosen. But looks like this file has some syntax errors (my guess is line 16). Travis is reporting that it cannot parse the yaml file for this PR. Can you please take another look? 

https://travis-ci.org/firebase/firebase-admin-python/requests